client/src/main/java/com/paypal/selion/internal/platform/grid/AbstractTestSession.java

```diff
@@ -151,10 +151,11 @@ protected void initializeAdditionalCapabilities(String[] additionalCapabilities,
 
     protected final Map<String, Object> parseIntoCapabilities(String[] capabilities) {
         Map<String, Object> capabilityMap = new HashMap<String, Object>();
+        final int TWO = 2;
         for (String eachCapability : capabilities) {
             // split into key/value at the ':' character
             String[] keyValuePair = eachCapability.split(":", 2);
-            if (keyValuePair.length == 2) {
+            if (keyValuePair.length == TWO) {
```

Comment:
this is just silly.. the original comparison is fine ... we should add an exception for this one.